src/core_plugins/timelion/public/directives/timelion_interval/timelion_interval.html

```diff
@@ -5,6 +5,7 @@
   ng-class="{ 'timelion-interval-custom-compact': interval === 'other' }"
   ng-model="otherInterval"
 ><select
+  aria-labelledby="timelionInterval"
```

Comment:
You could use `for` on the label instead. Which is imho also a little nicer in case of what the directive relies on. The `aria-labelledby` way the directive relies on being used in a context where a label with that given `id` exists. If you use `for` the directive just renders something with an `id` and the place where it is used utilize that `id` for its `label`.

Also I think this directive is used inside of timelion where there isn't a label currently. Since we still might want to use that directive in both places and make it accessible in both places, in this case we might be better adding an `aria-label` to it, that holds the label directly, without requiring the user of the directive to label it correctly.